install/agones/README.md

```diff
@@ -0,0 +1,83 @@
+# Agones
+
+[Agones](https://agones.dev) is designed as a batteries-included, open-source, dedicated game server hosting and scaling project built on top of Kubernetes, with the flexibility you need to tailor it to the needs of your multiplayer game.
```

Comment:
We now have this twice, seems like we shouldn't.

Reply:
Ok, I will remove it and change the title of the document for `Install Agones using Helm`